SPEC.md

```diff
@@ -2260,17 +2255,52 @@ Example 2:
   task example {
   File input_file = "my_input_file.bam"
   String output_file_name = sub(input_file, "\\.bam$", ".index") # my_input_file.index
-  
+
   command {
     echo "I want an index instead" > ${output_file_name}
   }
-  
+
   output {
     File outputFile = ${output_file_name}
   }
 }
 ```
 
+## Array[Int] range(Int)
+
+Creates an array of integers of length equal to the range argument. For example `range(3)` provides the array: `(0, 1, 2)`.
```

Comment:
TOL: as I mentioned to @ruchim, the previous library sections have descriptions that follow this formula: "Given X, this returns Y". I'd suggest following this model, but it's not critical.